File: wear/src/main/java/hibernate/v2/testyourandroid/ui/activity/InfoAndroidVersionActivity.java
package hibernate.v2.testyourandroid.ui.activity;

import android.os.Bundle;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;

import hibernate.v2.testyourandroid.R;
import hibernate.v2.testyourandroid.ui.fragment.InfoAndroidVersionFragment;

public class InfoAndroidVersionActivity extends BaseActivity {

	@Override
	public void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		setContentView(R.layout.activity_container_no_drawer);

		Fragment fragment = new InfoAndroidVersionFragment();
		FragmentManager fragmentManager = getSupportFragmentManager();
		fragmentManager.beginTransaction()
				.replace(R.id.container, fragment)
				.commit();
	}

}

File: wear/src/main/java/hibernate/v2/testyourandroid/ui/info/InfoAndroidVersionActivity.kt
package hibernate.v2.testyourandroid.ui.info

import android.os.Bundle
import hibernate.v2.testyourandroid.ui.base.BaseActivity

class InfoAndroidVersionActivity : BaseActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        initFragment(InfoAndroidVersionFragment())
    }
}